edx_proctoring/templates/proctoring/seq_proctored_exam_footer.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+<div class="faq-proctoring-exam">
+  <h4> {% trans "See Also" %} </h4>
+    <p>
+    {% blocktrans %}
+      <a class="footer-link" href="#">
+        Frequently asked questions about proctoring and earning college credit.
+      </a>
+      <a class="footer-link" href="#">
+        Technical Requirements for taking a proctored exam
```

Comment:
"Technical requirements for proctoring software" goes to:
http://clientportal.softwaresecure.com/support/index.php?/Knowledgebase/Article/View/252/0/system-requirements-remote-proctor-now